File: asignaturas/sql/Asignaturas-IvanRonco/Profesores_Y_Modulos_BD-Java/src/main/java/dto/modulo/ModuloDto.java
package dto.modulo;

import com.squareup.moshi.Json;
import lombok.AllArgsConstructor;
import lombok.Data;
import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Data
@AllArgsConstructor
@Json(name = "modulo")
@Root(name = "modulo")
public class ModuloDto {
    @Json(name = "modulo_id")
    @Element(name = "modulo_id")
    String uuid;

    @Json(name = "nombre")
    @Element(name = "nombre")
    String nombre;

    @Json(name = "curso")
    @Element(name = "curso")
    String curso;

    @Json(name = "grado")
    @Element(name = "grado")
    String grado;
}

File: asignaturas/sql/SergioDeIscar-Asignatura-Kotlin/SergioDeIscar-Asignatura-Kotlin/src/main/kotlin/dto/ModuloDto.kt
package dto

import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "modulo")
data class ModuloDto(
    @field:Attribute(name = "uuid")
    @param:Attribute(name = "uuid")
    val uuid: String,

    @field:Element(name = "nombre")
    @param:Element(name = "nombre")
    val nombre: String,

    @field:Element(name = "curso")
    @param:Element(name = "curso")
    val curso: String,

    @field:Element(name = "grado")
    @param:Element(name = "grado")
    val grado: String
)